### PR TITLE
Clarification for adding multiple sticker packs

### PR DESCRIPTION
**Pull request by MarvinJWendt**
There was no description that the `identifier` must be the same as the diretory name. I changed this to clarify the use of it.

PS: I haven´t edited the last line, but GitHub says so. Probably the line ending changed for that line as I edited the file in GitHub Web.

### CODE REVIEW

**Comment by do7man** [comment]
وي هيا شاهي بخار ☕️

File: android/README.md [hunk 1]
```diff
@@ -138,4 +138,4 @@ The result is in a row corresponding to the column named `result`. The value wil
 
 Note that a pack can be added to either the main WhatsApp app, WhatsApp Business, or both. It is recommended you continue to present a button to add the pack to WhatsApp if the sticker pack is not added to one or more of the apps. Refer to the class [WhitelistCheck](app/src/main/java/com/example/samplestickerapp/WhitelistCheck.java) for sample logic.
 
-Your app can only query whether the packs it provides have been added and it can't check for information about sticker packs from other apps.
\ No newline at end of file
+Your app can only query whether the packs it provides have been added and it can't check for information about sticker packs from other apps.
```
Comment (simonzhexu) [inline android/README.md]: what changed here?

File: android/README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ If you would like to create a sticker app using the sample app, you only have to
 * Navigate to SampleStickerApp/app/src/main/assets in Android Studio. 
 * Inside the assets folder, folder 1 contains a number of sample sticker art files. Replace these with your own sticker files.
 * Also replace the sample tray icon PNG with your own tray icon. 
-* If you'd like to have more than 1 sticker pack in your app, simply create a folder named "2" or "3", etc. within the assets folder and place your art and tray icon in there. 
+* If you'd like to have more than 1 sticker pack in your app, simply create a folder named "2" or "3", etc. within the assets folder and place your art and tray icon in there. Make sure to add your new pack into your `contents.json` by duplicating the existing pack data and changing the `identifier` to the new directory name. (For example change `identifier` to "2" if your new pack assets are in the directoy "2")
```
Comment (simonzhexu) [inline android/README.md]: Looks good. Please resolve the conflict, and this diff is good to go.

**Comment by simonzhexu** [comment]
I renamed the folder, if you have trouble resolving the conflicts, feel free to open a new pull request from master and send it.

File: android/README.md (hunk 1 above)
Comment (MarvinJWendt) [reply android/README.md]: I actually don´t know but I think the line ending changed because I edited it in GitHub Web. So there´s no visible change.

**Pull request merged**